vocabularies/UI.xml

```diff
@@ -1601,6 +1601,31 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         <Annotation Term="Core.Description" String="The contents of this property must not be propagated to the app-to-app navigation context" />
       </Term>
 
-    </Schema>
+      <Term Name="UserInteraction" Type="UI.UserInteractionType" AppliesTo="NavigationProperty">
+        <Annotation Term="Core.Description" String="When the annotated navigation property occurs in a response, the UI shall use the corresponding collection to immediately interact with the user and then repeat the corresponding request with the additional information obtained from the user"/>
+        <Annotation Term="Core.LongDescription" String="A navigation property thus annotated may be interpreted as not belonging to the object model but containing auxiliary information that is not always needed."/>

```

Comment:
Could you please describe the purpose / use case for this annotation?

Reply:
See https://wiki.wdf.sap.corp/wiki/display/odata/Remedial+action

Reply:
The vocabulary is on public GitHub, so we need a publicly visible description, either as part of the term definition, or linked from the term definition.

Reply:
I have described a use case in the example file.